### PR TITLE
TC07 - User can create new account

### PR DESCRIPTION
**Pull request by sattdn21-01-02**

### CODE REVIEW

File: src/test/java/com/logigear/RegisterTest.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    public void TC07() {
+        LoggerHelper.startTestCase("TC07 - User can create new account");
+        homePage.goToRegisterPage();
+
+        String password = DataHelper.getRandomValidPassword();
+        Account account = new Account(DataHelper.getRandomValidEmail(),
+                password, DataHelper.getRandomValidPid());
+        registerPage.register(account);
+
+        String actualMsg = Constant.REGISTER_CONFIRM_MSG;
```
Comment (fey031) [inline src/test/java/com/logigear/RegisterTest.java:29]: if you define this message in Constant, don't need declare new variable for it

File: src/test/java/com/logigear/RegisterTest.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        String expectedMsg = registerPage.getSuccessfulMessage();
+        Assert.assertEquals(actualMsg, expectedMsg);
```
Comment (fey031) [inline src/test/java/com/logigear/RegisterTest.java:31]: missing assert message

Comment (sattdn21-01-02) [reply src/test/java/com/logigear/RegisterTest.java:31]: Thanks for your review. I just have fixed and pushed into this branch. Please review my newest update.

File: src/test/java/com/logigear/RegisterTest.java (hunk 1 above)
Comment (sattdn21-01-02) [reply src/test/java/com/logigear/RegisterTest.java:29]: Thanks for your review. I just have fixed and pushed into this branch. Please review my newest update.

File: src/test/java/com/logigear/RegisterTest.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+    @Test(description = "TC07 - User can create new account")
+    public void TC07() {
+        LoggerHelper.startTestCase("TC07 - User can create new account");
+        homePage.goToRegisterPage();
+
+        String password = DataHelper.getRandomValidPassword();
+        Account account = new Account(DataHelper.getRandomValidEmail(),
+                password, DataHelper.getRandomValidPid());
+        registerPage.register(account);
```
Comment (LoliSlayerMC) [inline src/test/java/com/logigear/RegisterTest.java:25]: why did you create variable for password only? Should be consistent, create variables for all the data or neither all.

**Review by LoliSlayerMC — CHANGES_REQUESTED** [review]

File: src/test/java/com/logigear/RegisterTest.java (hunk 3 above)
Comment (sattdn21-01-02) [reply src/test/java/com/logigear/RegisterTest.java:25]: Thank you for the review, I will edit it now

**Review by LoliSlayerMC — APPROVED** [review]
LGTM

**Pull request merged**